### ISSUE TITLE
ASP.NET WebAPI 2 Error Processing Queue Object Reference Exception

### ISSUE THREAD

**Issue by mlturner88** [frhagn/Typewriter#116]
We just started a new project using Angular 1.x and Web API 2.x and have installed your lovely extension. On my colleagues machine we are able to generate everything that we need perfectly but unfortunately we are unable to get the template rendering to work on other machines. I am getting the below output when I attempt to. Having a look at [the RoslynFileMetadata constructor](https://github.com/frhagn/Typewriter/blob/master/src/Roslyn/RoslynFileMetadata.cs) I can only assume one of the objects being initialized is returning null. Please advise! Thanks in advance.

```
ERROR: Error processing queue: Object reference not set to an instance of an object.
   at Typewriter.Metadata.Roslyn.RoslynMetadataProvider.GetFile(String path)
   at Typewriter.Generation.Controllers.GenerationController.<>c__DisplayClass5_0.<OnTemplateChanged>b__1()
   at Typewriter.Generation.Controllers.EventQueue.ProcessQueue()
```

**Comment by frhagn** [comment]
Hi @mlturner88, 
I haven't seen this problem before. Are you running Visual Studio 2015 Update 2? Try to uninstall Typewriter, restart Visual Studio and install Typewriter again.

**Comment by mlturner88** [comment]
My apologies I should have mentioned that before. I am using VS2013 Update 5. I've tried reinstalling Typewriter a few times now without any success, unfortunately.

**Comment by frhagn** [comment]
Ah ok that's wierd, vs 2013 should use the CodeDomMetadataProvider, not Roslyn... Have you installed some kind of Roslyn preview for vs 2013 or something like that? Also, is your colleague running 2013 update 5?

**Comment by mlturner88** [comment]
I haven't to my knowledge. My colleague is using the same version as me, yes. The main difference is I have the Xamarin tools installed and he doesn't.

**Comment by frhagn** [comment]
Look for "Roslyn Preview" in Extensions and Updates.. If it's there, try to disable it and restart Visual Studio

**Comment by mlturner88** [comment]
I'm not seeing that installed. I just updated to 1.7.3 as well with no luck.

**Comment by frhagn** [comment]
I just published a new version (1.7.4) which hopefully fixes this issue.

**Comment by mlturner88** [comment]
That fixed it. Thank you very much!
